routes/socket/user-events.js

```diff
@@ -828,21 +828,24 @@ module.exports.handleAddNewGame = (socket, passport, data) => {
 };
 
 /**
+ * @param {object} socket - user socket reference.
  * @param {object} passport - socket authentication.
  * @param {object} game - target game.
  * @param {object} data - from socket emit.
  */
-module.exports.handleAddNewClaim = (passport, game, data) => {
+module.exports.handleAddNewClaim = (socket, passport, game, data) => {
 	const playerIndex = game.publicPlayersState.findIndex(player => player.userName === passport.user);
 
+	if (!/^wasPresident|wasChancellor|didSinglePolicyPeek|didPolicyPeek|didInvestigateLoyalty$/.exec(game.private.seatedPlayers[playerIndex].playersState[playerIndex].claim)) {

```

Comment:
fyi this may not be exactly as you think as I don't think `^` persists through `|` but its probably fine regardless.

Reply:
Hmm, you are correct it seems. Good to know - it shouldn't matter for this specific use, but for future use.